content/about-us/involved/index.en.md

```diff
@@ -15,20 +15,20 @@ We welcome your contributions, below are handful of ideas 💜
 
 - Update [R-Ladies directory](/directory) with your details 
 - Join our [community slack](https://rladies.org/form/community-slack) channel
-- Follow us over social media. Periodically we share about upcoming activities
-- Join your regional R-Ladies chapter or the remote chapter, perfect space to connect with likeminded individuals
+- Follow us over [social media](https://hachyderm.io/@RLadiesGlobal). Periodically we share about upcoming activities
+- Join your regional [R-Ladies chapter](https://www.meetup.com/pro/rladies/) or the remote chapter, perfect space to connect with likeminded individuals
 - Attend the online/in-person events. The details for upcoming events could be found over slack channels/ social media
-- Content along with recordings for our past events are openly accessible over YouTube channel, reach out to speakers that you like and let them know that you appreciate their content
+- Content along with recordings for our past events are openly accessible over [YouTube](https://www.youtube.com/@RLadiesGlobal) channel, reach out to speakers that you like and let them know that you appreciate their content
 
 #### 💡Always room for more 
 
 - Propose a talk for a meetup/event/conference 
-- Become a chapter organizer by volunteering with your local chapter or starting your own (please get in touch via chapters[at]rladies[dot]org)
-- Contribute a blog post to the website, we are always here to help. 
+- Become a chapter organizer by volunteering with your local chapter or starting your own (please get in touch via [chapters@rladies.org](mailto:chapters@rladies.org))
+- Contribute a blog post to the [website](https://rladies.org/), we are always here to help. 
 
 #### 🤝 Above and beyond 
 
-- Join the global team, help us create a diverse space 
-- Create issues or PR(s) over our guide or website when you see things that can be improved
+- Join the [global team](https://rladies.org/about-us/global-team/), help us create a diverse space 
+- Create issues or PR(s) over [GitHub](https://github.com/rladies) for our [website](https://github.com/rladies/rladies.github.io) & [guide](https://github.com/rladies/rladiesguide) when you encounter sections which could be improved
 - Contribute to community discussion and share your thoughts
-- Support your local chapters with venue space and funding for food and beverages
+- Support your local chapters with venue space and [funding](https://www.paypal.com/donate/?hosted_button_id=VB4EK67MJ373E) for food and beverages
```

Comment:
That PayPal link is from RLadies Global account. We can't receive money for specific chapters there. But maybe we can add it in a different point about supporting our general infrastructure?